.github/workflows/pull-requests.yml

```
@@ -0,0 +1,19 @@
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  codebase-tests:
+    uses: ./.github/workflows/codebase-testing.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/codebase-testing.yml": open /home/jailuser/git/.github/workflows/codebase-testing.yml: no such file or directory

(workflow-call)